rosie.py

```diff
@@ -1,49 +1,64 @@
-from sys import argv, exit
+"""Rosie.
 
+Usage:
+  rosie.py run (chamber_of_deputies | federal_senate)
+  rosie.py run (chamber_of_deputies | federal_senate) [--years <years>]
+  rosie.py run (chamber_of_deputies | federal_senate) [--path <path>]
+  rosie.py run (chamber_of_deputies | federal_senate) [--path] [--years <years>]
+  rosie.py test [chamber_of_deputies | federal_senate]
 
-def entered_command(argv):
-    if len(argv) >= 2:
-        return argv[1]
-    return None
+Options:
+    --years   runs Rosie fetching data and finding suspicions for a specific set of years
+    --path    directory in which Rosie saves fetched data and the suspicions CSV [default: /tmp/serenata-data]
 
+"""
 
-def help():
-    message = (
-        'Usage:',
-        '  python rosie.py run chamber_of_deputies [<path to output directory>]',
-        'Testing:',
-        '  python rosie.py test',
-        '  python rosie.py test chamber_of_deputies',
-    )
-    print('\n'.join(message))
+from sys import exit
+from docopt import docopt
 
 
-def run():
+def main():
+    arguments = docopt(__doc__)
+
+    if arguments['run']:
+        run(**arguments)
+
+    if arguments['test']:
+        return test(**arguments)
+
+
+def run(**args):
     import rosie
     import rosie.chamber_of_deputies
     import rosie.federal_senate
 
-    if len(argv) >= 3:
-        target_module = argv[2]
-    else:
-        print('A module must be provided.')
-        help()
-        exit(1)
-    target_directory = argv[3] if len(argv) >= 4 else '/tmp/serenata-data/'
+    target_directory = args['<path>'] if args['--path'] else '/tmp/serenata-data/'
+
+    commands = ('chamber_of_deputies', 'federal_senate')
+    target_module, *_ =  filter(lambda x: arguments.get(x), commands)
+
     klass = getattr(rosie, target_module)
-    klass.main(target_directory)
 
+    if args['--years']:
+        years = args['<years>']
+        years = [int(num) for num in years.split(',')]
+        klass.main(target_directory, years=years)
+    else:
+        klass.main(target_directory)
```

Comment:
No need for this `if`/`else` here since:

* if `args['--years']` is `None`
* and considering that the default value for `years` is `None` (and it is)
* then `klass(target_dir)` is exactly equal to `klass(target_dir, args['--years']`

However I'm afraid we might need to convert the string `2016,2017` to a list of integers, i.e. `[2016, 2017]`. For example:

```python
if args['--years']:
   args['--years'] = [int(v) for v in args['--years'].split(',')]

klass.main(target_directory, args['--years'])
```